### DISCUSSION TITLE
Reconsider adding the 2 min trinket for all

### DISCUSSION THREAD

**Discussion by minthes** [chromiecraft/chromiecraft#4690] [Suggestions]
6 months have passed since this was last suggested and I believe the community changed its stance on the 2 minute trinket for all. I think the poll should be done again.

**Comment by HasieEST** [comment]
6 months has passed, most likely 6 months will pass before 70 bracket, where 2 min PvP trinkets actually become available. It is understandable, that having lower cooldown for PvP trinkets alters gameplay loop fair bit, but it does not impact actual PvP that much in my humble opinion. There are currently more pressing issues regarding PvP, that have much bigger impact on overall PvP environment, than lowering PvP trinket cd to 2 min, which is a placebo buff at best, if it went through. For example double dipping on damage reductions(serpent shot - chimera shot comes to mind right away),  "unintended" nerfs to HoTs and bandages, for some reason nerfing mage evo glyph(in my humble opinion, kinda BS). As there has been a lot of discussion about that already some people have opted to not do PvP at all for 6 months. Bringing 2 min trinkets forward by 6 months wont solve the core issues.
P.S. Did a bit of digging, only non-resilience PvP trinket, that is available on blizzard hosted servers, have 5 min cd, once you get resilience on them, their cd gets lowered to 2 min. There is Insignia of PvP Pwn but there is no actual way to obtain it nor it should be imo.

**Reply by minthes** [reply]
I think this is a false dichotomy. 
It's not either we fix Y or Z, both can be fixed and fixing one doesn't mean we can't fix the other.
![image](https://user-images.githubusercontent.com/94015041/209923272-c2e24b18-b8b0-4740-b930-88e52a5c9ccd.png)

And if something isn't done, we're gonna have to deal with ^ for 6 more months or two more content phases.

**Reply by HasieEST** [reply]
I didn't intent to create a dichotomy between Trinket and PvP tuning issues, if that happened, it is my mistake. I merely tried to draw parallels on the impact of the Trinket cooldown alteration and altering tuning numbers/interactions which are tuned in order to create a bit more balanced environment. 

Since I had to dig through discord to find reasonings why people are for it, I'll list them down here for future reference for anyone wishing to give their thoughts on the matter. I will add my on thoughts afterwards, trying to convey how I perceive them and what I think about them. In addition I'll try to add more points, which would either support or be against the idea of introducing this suggested alteration to the server. I would like to reiterate, that I personally do not feel like the change would be necessary.

As I commented before hand, I understand that there is a theoretical impact on higher cd trinket compared to lower cd trinket and removing that aspect right now would not hurt anyone. That is unarguable point that needs not to be overturned.

As Vitus pointed out in Discord, there is legitimate justification for this type of change, due already existing alterations made for battlegrounds *Damage/Healing nerfs*.  I agree with him, there is justification for it, hence why some people are discussing about this.

He also pointed out that we are using WoTLK talents and spells, not on Vanilla nor TBC. What he wanted to say with that, I have no real clue beyond, hey these are the spells and these are the talents. Perhaps we could get an elaboration on it, but at the time of writing that is how I understand it.

There is some suggestion that people will roll Horde, if PvP trinkets were 2 minutes. Credit where it is due, there is definitely a portion of people, who choose to be human due to their access to very powerful PvP racial. However, I do no think that with addition of 2 minute trinket 6 months prior to their true timeline release _70 with arena points_  has major impact. There is a way however to confirm or deny this hypothesis, by conducting a sizeable survey on why people choose their race and how much did Human racial impact/not impact their choice of race. Sample size needs to be decently sized, let say 300-400 unique players (aka 1 player regardless, if they have 1 or many characters/ 1 or 5 accounts ) to have a legitimate and justified backing for the statement.

Minthes has kindly posted a snipped of CC PvPStats, showcasing top 10 players, that have won within this month. This prompted me to do a lil check on Top100 players. Out of that I choose 72 characters, that were above lvl 60 and see how many humans are there and how many of them are not humans,  who could be humans. Results: 28 Human Characters, 44 Non Human Characters,  26 characters, which could have been Humans. With this little sample Size, we can say that there is somewhat balance between characters, which could be humans choosing to be other races. However this little sample doesn't paint whole picture and, if possible, would require larger data sample in order to give more educated input about general human majority over non human counterparts for 6 classes, which humans can be, thus I would not say that majority of people, who are actively PvP, are Humans purely due to strong Racial. 
Link for anyone wishing to see spreadsheet and expand on it: `https://docs.google.com/spreadsheets/d/1dBMzzw-xN5PAGtIwgWse5EvdrK2DvjeBz0x8eTowRqU/edit?usp=sharing`
Additionally I would point out that cherry picking a very small sample size isn't constructive argument towards reasoning why do A, B or C but I digress.

First thing that comes to mind, when implementing this solution is the aftermath of 70 bracket and the availability of 2 minute trinkets with resilience. Giving stuff to people is easy, taking them away is much harsher though. Additionally, how can person be eligible for 2 minute trinket. Even in Current Classic Wrath Servers, everyone below 70 have only access to 5 minute CD Trinkets. By giving people 2 minute trinkets now, there is a decent chance that once 70 bracket hits and staff decides to revert this change, people would be very bummed out, if not even outraged, for losing access to cheap 2 min trinket without resilience.  Once open, Pandoras box never closes.

These are some of my thoughts and citations from Discord about this matter.  
TL;DR I believe, there is net positive to gain from this change, but I do not think this change is necessary and could cause further headache later down the line, if wished to be reverted. Suggestion lacks clearer criterium for who and how you gain access to lower CD trinket, if it becomes available. There is a lack of conclusive evidence, that majority of active PvP community is Human, and when they are human, it is purely due to Racial.
